compiler/crates/relay-compiler/tests/relay_compiler_integration/fixtures/preloadable_query_flow.expected

```diff
@@ -0,0 +1,129 @@
+==================================== INPUT ====================================
+//- foo.js
+graphql`
+  query fooQuery @preloadable {
+   userName
+  }`;
+
+//- relay.config.json
+{
+  "language": "flow",
+  "schema": "./schema.graphql",
+  "eagerEsModules": true,
+  "persistConfig": { 
+    "file": "./operations.json" 
+  }
+}
+
+//- operations.json
+{}
```

Comment:
@tobias-tengler somehow the query text is missing in the operations - this is maybe related to the way the integration test prints these files...

Reply:
This is the input. The local persister throws if the file doesn't already exists. This is just here to satisfy the persister.
<img width="579" alt="Screenshot 2024-01-08 at 22 23 20" src="https://github.com/facebook/relay/assets/45513122/fb0f11f5-9f2a-40fe-8f10-35e7cb08f06f">